client/rest-high-level/src/main/java/org/elasticsearch/client/FreezeIndexRequest.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Licensed to Elasticsearch under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.elasticsearch.client;
```

Comment:
I agree that we should not put them here. I asked some people to create a core package, but I think thats wrong now, since all of our clients have their own converters / client, and eventually should be in their own package. So its safe to assume that we should have a indices package under client, and we may have to move the existing responses here for things like count etc.. that are in the core package already. That was totally my mistake 🤦‍♂️